### PR TITLE
Remove mention of `contact@` and the slack

### PR DESCRIPTION
**Pull request by laanwj**
The `contact@` address for "For general enquiries and press" is unattended. No one is handling these since @btcdrak became inactive, so imo it's better to remove it.

As far as I know, no bitcoin core developers regularly attend the Slack, so remove it from contact information as well.

### CODE REVIEW

**Comment by luke-jr** [comment]
I recently rejoined the slack and noticed at least a couple other devs there, but I support removing it anyway. (cc @achow101)

Concept ACK

**Comment by gmaxwell** [comment]
utACK!

**Comment by MeshCollider** [comment]
utACK, I'm in the slack but literaly never log in and check it

**Comment by MarcoFalke** [comment]
Could remove the slack from `_includes/references.md`?

**Comment by jnewbery** [comment]
Concept ACK. There are also a bunch of slack links in various release announcements. Should we also remove those?

**Comment by laanwj** [comment]
> Could remove the slack from `_includes/references.md`?

Agree.

> Concept ACK. There are also a bunch of slack links in various release announcements. Should we also remove those?

At least for the code repository we have a policy, more or less, not to change historical announcements. But don't really have a strong opinion in this case.

**Comment by jnewbery** [comment]
> At least for the code repository we have a policy, more or less, not to change historical announcements.

Right - I'm refering to the release *announcements* (eg https://bitcoincore.org/en/2017/10/02/release-0.15.0.1/) rather than the release *notes* which are copied from the repo (eg https://bitcoincore.org/en/releases/0.15.0.1/).

I don't think we *need* to remove those links, but I have a slight preference to get rid of them.

**Comment by laanwj** [comment]
I've removed all references to slack.

**Comment by MarcoFalke** [comment]
ACK 762cb76a48e5807e274e53f91d56e47309959e4f

**Comment by jnewbery** [comment]
ACK 762cb76a48e5807e274e53f91d56e47309959e4f.

Can't verify the chinese/japanese, but otherwise looks good. Thanks!

**Comment by fanquake** [comment]
utACK 762cb76

File: _posts/en/posts/2016-03-15-announcelist.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ version: 2
 ---
 In an effort to increase communications, we are launching an opt-in, _announcement-only_ mailing-list for users of Bitcoin Core to receive notifications of security issues and new releases.
 
-While the Bitcoin Core project has a variety of communication channels, this website, Twitter [@bitcoincoreorg][], [Slack][] there have been a number of requests for an [announcement mailing list](/en/list/announcements/join). It is intended to be extremely low volume, focused on critical notifications as well as to announce new releases.
+While the Bitcoin Core project has a variety of communication channels, this website, Twitter [@bitcoincoreorg][], there have been a number of requests for an [announcement mailing list](/en/list/announcements/join). It is intended to be extremely low volume, focused on critical notifications as well as to announce new releases.
```
Comment (harding) [inline _posts/en/posts/2016-03-15-announcelist.md]: Nit: extraneous comma after "Twitter [@bitcoincoreorg][],"

File: _posts/ja/posts/2017-11-11-release-0.15.1.md [hunk 2]
```diff
@@ -80,7 +80,7 @@ GUIの設定は`-resetguisettings`引数が使われた際、設定が消去さ
 
 詳細については[リリースノート][]を参照ください。ダウンロードは[ダウンロードページ][]からできます。
 
-ご不明な点がありましたら、[IRC][]もしくは[Slack][]のチャットルームまでご連絡ください。
+ご不明な点がありましたら、[IRC][]もしくはのチャットルームまでご連絡ください。
```
Comment (harding) [inline _posts/ja/posts/2017-11-11-release-0.15.1.md]: This doesn't seem to be quite right.  Google Translate shows it as "If you have any questions, please contact IRC or a chat room."  However, it seems to be a fairly minor problem and an old announcement, so I think we can ignore it.

**Pull request merged**